### DISCUSSION TITLE
Can not connect to local host

### DISCUSSION THREAD

**Discussion by Gottano** [reflex-dev/reflex#981] [Q&A] [answered]
When I open localhost (http://localhost:3000/) I get "connection refused" error message.  I tried a couple of browsers and get the same error.  

Any ideas?

**Comment by Lendemor** [comment]
Please give more informations.
What is your OS?
Are you running `pc run` correctly (any error?) ?
Can you check if the process is correctly listening to port 3000 ?

**Reply by Gottano** [reply, marked as answer]
Thank you for your suggestions. It turns out that one of the folders had an exclamation mark as part of its name, which threw up errors I hadn't noticed. I renamed the folder and is running fine now.
